inline-caching/cache-loader/src/main/java/io/vmware/event/ItemAsyncEventListener.java

```diff
@@ -0,0 +1,84 @@
+package io.vmware.event;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.SQLDialect;
+import org.jooq.Table;
+import org.jooq.impl.DSL;
+
+import org.apache.geode.LogWriter;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.Operation;
+import org.apache.geode.cache.asyncqueue.AsyncEvent;
+import org.apache.geode.cache.asyncqueue.AsyncEventListener;
+
+
+@SuppressWarnings("deprecation")
+public class ItemAsyncEventListener implements AsyncEventListener {
+
+	private static LogWriter log;
+	
+	static {
+		log = CacheFactory.getAnyInstance().getDistributedSystem().getLogWriter();
+	}
+
+	@Override
+	public boolean processEvents(List<AsyncEvent> events) {
+		try(Connection conn = ConnectionPool.getInstance().getConnection()) {
+			log.info("connected:"+ conn.getSchema());
+			return process(conn, events);
+		} catch (SQLException e) {
+			log.error("Exception", e);
+			return false;
+		}
+	}
+
+	private boolean process(Connection conn, List<AsyncEvent> events) {
+		DSLContext create = DSL.using(conn, SQLDialect.POSTGRES);
+
+		for (AsyncEvent ge : events) {
+			Operation operation = ge.getOperation();
+			if(Arrays.asList(Operation.UPDATE, Operation.CREATE, Operation.LOCAL_LOAD_CREATE).contains(operation)) {
+				createOrUpdate(create, ge);
+			} else {
+				log.error("operation not implemented: " + operation);
+			}
+		}
+
+		return true;
+	}
+
+	private static void createOrUpdate(DSLContext create, AsyncEvent ge) {
+		int itemId = Integer.parseInt((String) ge.getKey());
+		String value = (String) ge.getDeserializedValue();
+		log.info(String.format("process event itemId: %s value: %s", itemId, value));
+
+		Table<Record> table = DSL.table("pgbench_tellers");
+		Field<Object> filler = DSL.field("filler");
+		Field<Object> tid = DSL.field("tid");
```

Comment:
I wonder if a little more descriptive variable names would make a difference here... maybe `tellerTable` and `fillerField` and `idField`

Reply:
renamed